README.md

```diff
@@ -30,11 +30,19 @@ Type git clone git@github.com:letarulinda/mobile-vesrion-setup.git and hit enter
 
 - You can now view the HTML document in your preferred browser.
 
-## Author Letaru Linda Masia
+## Authors
+
+👤 **Letaru Linda Masia**
 
 - GitHub: https://github.com/letarulinda
 - LinkedIn: www.linkedin.com/in/linda-letaru-masia-7b48a9175
 
+👤 **Adel Guitoun**
+
+- GitHub: [@adel-gu](https://github.com/adel-gu)
+- Twitter: [@GuitounAdel](https://twitter.com/@GuitounAdel)
+- LinkedIn: [adelguitoun](https://linkedin.com/in/adelguitoun)
+
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] [optional] Kindly ensure you have a MIT license,so that users are not presented with 404 page when they click to view your licence.